File: core/src/ru/myitschool/cubegame/ai/DummyAI.java
package ru.myitschool.cubegame.ai;

import ru.myitschool.cubegame.entities.Enemy;
import ru.myitschool.cubegame.entities.Entity;
import ru.myitschool.cubegame.entities.enemies.GoblinWarrior;
import ru.myitschool.cubegame.screens.DungeonScreen;

/**
 * Created by Voyager on 11.12.2017.
 */
public class DummyAI extends AI {

    public DummyAI(Entity controlledEntity) {
        super(controlledEntity);
    }

    @Override
    public void aiAnalyze() {

    }
}

File: core/src/com/wolg_vlad/dcrawler/entities/ai/DummyAI.kt
package com.wolg_vlad.dcrawler.entities.ai

import com.wolg_vlad.dcrawler.entities.Entity

/**
 * Created by Voyager on 11.12.2017.
 */
class DummyAI(controlledEntity: Entity) : AI(controlledEntity) {
    override fun aiAnalyze() {}
}